techempower/scripts/perf/tfb-run.sh

```diff
@@ -26,6 +26,8 @@ pushd ".." > /dev/null
 LOGFILE="${PWD}/setup.log"
 K_CPU=6
 K_MEM=8192
+DEPLOYMENT_NAME="tfb-qrh-sample-0"
```

Comment:
@kusumachalasani - tfb database is not started after deletion. Also can we change the port number as prometheus too runs on the same port?  

```
Deploying the application...
Removing the tfb instances...Unable to listen on port 9090: Listeners failed to create with the following errors: [unable to create listener: Error listen tcp4 127.0.0.1:9090: bind: address already in use unable to create listener: Error listen tcp6 [::1]:9090: bind: address already in use]
error: unable to listen on any of the requested ports: [{9090 9090}]
deployment.apps "tfb-database" deleted
deployment.apps "tfb-qrh-sample-0" deleted
service "tfb-database" deleted
service "tfb-qrh-service-0" deleted
servicemonitor.monitoring.coreos.com "tfb-qrh-0" deleted
done
servicemonitor.monitoring.coreos.com/tfb-qrh-0 created
deployment.apps/tfb-qrh-sample-0 created
service/tfb-qrh-service-0 created
```

Reply:
Yes, I 'm passing it